samples/msal-node-samples/ExpressTestApp/src/AuthProvider.ts

```diff
@@ -284,11 +282,11 @@ export class AuthProvider {
 
         } catch (error) {
             // in case there are no cached tokens, initiate an interactive call
-            if (error instanceof InteractionRequiredAuthError) {
+            if (error instanceof InteractionRequiredAuthError || error instanceof ClientAuthError) {
```

Comment:
What is the `ClientAuthError` condition used for?

Reply:
Currently [msal-node is masking the actual error from AAD](https://github.com/AzureAD/microsoft-authentication-library-for-js/issues/4878). So this is a temporary workaround until the issue gets resolved, although it's not ideal.

Reply:
I'm not a fan of putting temporary stuff in, particularly in docs/samples, I'd rather invest in fixing the bug

Reply:
I think @bmahal is already working on it -I'll revert this change and convert the PR to draft for now then. @bmahal let me know if there's anything I can help with debugging

Reply:
removed this as the fix is coming along